CHANGELOG.asciidoc

```diff
@@ -51,6 +51,7 @@ our agent are not vulnerable as well, as the SMTP appender was never used, this
 for exit spans - {pull}1788[#1788]
 * Only use emulated runtime attachment as fallback, remove the `--without-emulated-attach` option - {pull}1865[#1865]
 * Instrument javax.servlet.Filter the same way as javax.servlet.FilterChain {pull}1858[#1858]
+* Added support for naming spark transactions {pull}1894[#1894]
```

Comment:
How could we make sure people don't confuse this with Apache Spark™ suport?
Maybe like this?
```suggestion
* Added support for naming sparkjava (not Apache Spark) transactions {pull}1894[#1894]
```

Also relevant in supported-technologies.md